pypeit/core/extract.py

```diff
@@ -22,6 +22,7 @@
 from pypeit.core import pydl
 from pypeit.core import pixels
 from pypeit.core import arc
+from pypeit.core.gui import object_find as gui_object_find
```

Comment:
This import dependency should be removed.

Reply:
done